howto/update/upgrade-anbox.md

```diff
@@ -36,6 +36,14 @@ You can either run the package update manually or use the Juju command to run it
 
     juju exec --all -- /bin/sh -c 'sudo apt update && sudo apt upgrade -y'
 
+If the LXD charm is deployed on a machine that has a NVIDIA GPU installed, running the above command for the machine may upgrade the NVIDIA drivers, which accidentally suspends running containers with GPU support. Starting with the 1.17.1 release, the NVIDIA drivers are held from being upgraded until you upgrade the LXD charm by using the Juju command. To check the current installed NVIDIA drivers are held from being upgraded:
+
+    apt-mark showhold libnvidia-.*-$(nvidia-smi --query-gpu=driver_version --format=csv,noheader | cut -d'.' -f1)
```

Comment:
To avoid globbing (just in case):

```suggestion
    apt-mark showhold "libnvidia-.*-$(nvidia-smi --query-gpu=driver_version --format=csv,noheader | cut -d'.' -f1)"
```